cirq-core/cirq/sim/operation_target.py

```diff
@@ -86,9 +86,14 @@ def qubits(self) -> Tuple['cirq.Qid', ...]:
         """Gets the qubit order maintained by this target."""
 
     @property
-    @abc.abstractmethod
     def log_of_measurement_results(self) -> Dict[str, Any]:
         """Gets the log of measurement results."""
+        return {str(k): list(self.classical_data.get_digits(k)) for k in self.classical_data.keys()}
```

Comment:
[No change needed] This is OK, but it's pushing the upper limit of "amount of work allowed to be done in a `@property`."

Reply:
This is to support backwards compatibility. I moved this from `ActOnArgs` and `ActOnArgsContainer` to the base class `OperationTarget`.

Reply:
(Also I think we should deprecate this property next. Currently it's used in `SimulatorBase._run` to create the `cirq.Result`. However it would make more sense if we created a `cirq.Result` constructor/factory/subclass that accepted `ClassicalData` objects directly.)